pytest_pootle/env.py

```diff
@@ -0,0 +1,333 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+import shutil
+
+from django.utils.functional import cached_property
+
+
+class PootleTestEnv(object):
+
+    methods = (
+        "case_sensitive_schema", "content_type", "site_root", "system_users",
+        "permissions", "site_permissions", "tps", "languages", "vfolders",
+        "subdirs", "announcements")
+
+    def __init__(self, request):
+        self.request = request
+
+    @cached_property
+    def dirs(self):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [An attribute defined in pytest_pootle.env line 29 hides this method](https://www.codacy.com/app/dwayne/pootle/file/1968005668/issues/source?bid=1990595&fileBranchId=3109045#l26)